Dockerfile

```diff
@@ -10,5 +9,9 @@ COPY ./resources ./resources
 COPY ./src ./src
 COPY ./bin ./bin
 RUN npm run build
+
+FROM node:16-alpine
```

Comment:
this will run for both platforms and docker will push both images with a manifest, so that users do not have to specify platform themselves